client-js/app/index.html

```diff
@@ -19,7 +19,7 @@
   <!-- Add to homescreen for Chrome on Android -->
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="application-name" content="LabRAD">
-  <link rel="icon" sizes="192x192" href="/images/touch/chrome-touch-icon-192x192.png">
+  <link rel="icon" href="/images/labrad.png">
```

Comment:
I was going to say before it looks like the new LA clippers logo.
![image](https://cloud.githubusercontent.com/assets/8173925/13544031/00f83dd4-e226-11e5-9e74-3098a216392e.png)
 Now With Jim's color scheme it looks like it even more